v3/src/components/graph/adornments/count/count-registration.tsx

```diff
@@ -12,7 +12,8 @@ import { useDataConfigurationContext } from "../../hooks/use-data-configuration-
 const Controls = () => {
   const graphModel = useGraphContentModelContext()
   const dataConfig = useDataConfigurationContext()
-  const existingAdornment = graphModel.adornments.find(a => a.type === kCountType) as ICountModel
+  const adornmentsStore = graphModel.adornmentsStore
+  const existingAdornment = adornmentsStore.adornments.find(a => a.type === kCountType) as ICountModel
```

Comment:
There are a number of this kind of cast throughout the code. I suggest adding a `view` like this to the `AdornmentsStore`:
```typescript
    findAdornmentOfType<T extends IAdornmentModel = IAdornmentModel>(type: string): T | undefined {
      return self.adornments.find(adornment => adornment.type === type) as T
    }
```
and then clients can use:
```typescript
  const countAdornment = adornmentsStore.findAdornmentOfType<ICountModel>(kCountType)
```